### PR TITLE
add main uri

### PR DESCRIPTION
**Pull request by andreaturli**
@grkvlt could you please have a look?

### CODE REVIEW

File: software/nosql/src/main/java/brooklyn/entity/nosql/riak/RiakNodeSshDriver.java [hunk 1]
```diff
@@ -316,6 +325,9 @@ public void launch() {
             launchScript.environmentVariablesReset(newPathVariable);
         }
         launchScript.failOnNonZeroResultCode().execute();
+
+        final String mainUri = String.format("http://%s:%s/admin", entity.getAttribute(Attributes.HOSTNAME), entity.getAttribute(RiakNode.RIAK_WEB_PORT));
```
Comment (grkvlt) [inline software/nosql/src/main/java/brooklyn/entity/nosql/riak/RiakNodeSshDriver.java]: Does it need to be `final`?

File: software/nosql/src/main/java/brooklyn/entity/nosql/riak/RiakNodeSshDriver.java [hunk 2]
```diff
@@ -68,7 +77,7 @@ public RiakNodeImpl getEntity() {
     @Override
     public Map<String, String> getShellEnvironment() {
         MutableMap<String, String> result = MutableMap.copyOf(super.getShellEnvironment());
-        // how to change epmd port, according to 
+        // how to change epmd port, according to
         // http://serverfault.com/questions/582787/how-to-change-listening-interface-of-rabbitmqs-epmd-port-4369
         if (getEntity().getEpmdListenerPort() != null) {
             result.put("ERL_EPMD_PORT", "" + Integer.toString(getEntity().getEpmdListenerPort()));
```
Comment (grkvlt) [inline software/nosql/src/main/java/brooklyn/entity/nosql/riak/RiakNodeSshDriver.java]: Can you fix this so it doesn't have the `"" + ` part before the `Integer#toString(Integer)` call?

**Comment by andreaturli** [comment]
the error doesn't seem related: unable to find a free port at or above 8080 at [test](https://builds.apache.org/job/incubator-brooklyn-pull-requests/948/org.apache.brooklyn.camp$camp-server/testReport/junit/io.brooklyn.camp.rest.resource/PlatformRestResourceTest/startServer/)

**Comment by andreaturli** [comment]
@grkvlt can I merge it?

**Pull request merged**